main/main.cpp

```diff
@@ -195,11 +196,13 @@ void setup()
     ha_system_set_topic = mqtt_topic + "/" + mqtt_fn + "/system/set"; // for control over mqtt
     if (others_haa)
     {
-      ha_config_topic = others_haa_topic + "/climate/" + mqtt_fn + "/config";
+      //ha_config_topic = others_haa_topic + "/climate/" + mqtt_fn + "/config";
+      ha_config_topic = others_haa_topic + "/climate/hvac_" + getId() + "/config";
     }
     else
     {
-      ha_config_topic = "homeassistant/climate/" + mqtt_fn + "/config";
+      //ha_config_topic = "homeassistant/climate/" + mqtt_fn + "/config";
+      ha_config_topic = "homeassistant/climate//hvac_" + getId() + "/config";
```

Comment:
Double //

Reply:
Also change the ID of climate will make update fw create new entity in the Home Assistant.